### ISSUE TITLE
[REQUEST] Support for Multi Receiver for trainer signal

### ISSUE THREAD

**Issue by RealTadango** [FrSkyRC/ETHOS-Feedback-Community#2238]
We use a lot of sSpektrum transmitters on our club and i now use a Multi module in the SpektrumRX mode for a trainer signal. This is great because we have trainer Spektrum transmitters in our club present for teaching. It would be great to have this option in ETHOS also so i can use a ETHOS transmitter for everything.

**Comment by LuHaGH** [comment]
Yes, this would be really nice.

**Comment by RealTadango** [comment]
And also FrskyRX mode :)

**Comment by LuHaGH** [comment]
The Multi Module supports many receiver protocols. So if you add the serial input into the trainer mode all protocols should work.

**Comment by RealTadango** [comment]
No, i don't want to hack stuff like this. ETHOS needs to support the great thing that is MP. The current support is to limited. It is a good start but certainly needs to be improved.

**Comment by RealTadango** [comment]
Still very welcome

**Comment by RealTadango** [comment]
Still very welcome

**Comment by yak-54** [comment]
yes it is

**Comment by RealTadango** [comment]
Still needed

**Comment by amaury10** [comment]
I totally agree it's a useful feature. We are in the same situation in our club : lots of spektrum transmitters (and some of them replaced by EthOS radios) could be used as trainee radios.

**Comment by RealTadango** [comment]
Still needed

**Comment by Dsrenger** [comment]
I don't get the request. You are expecting FrSky to implement a reverse engineered Spektrum protocol into their hardware? 

Sorry, I am not a native speaker, just curious what gets requested here.

**Comment by RealTadango** [comment]
> I don't get the request. You are expecting FrSky to implement a reverse engineered Spektrum protocol into their hardware?
> 
> Sorry, I am not a native speaker, just curious what gets requested here.

No, i expect them to implement the multi trainer option from the Multi Protocol Module they already support....

**Comment by Dsrenger** [comment]
ah, didnt know this option of the MPM. Googled a little bit about it. Thanks for giving me this hint :)

have a nice day.

**Comment by RealTadango** [comment]
Still needed

**Comment by RealTadango** [comment]
Still wanted very much! @bsongis-frsky is this difficult to include? This would be perfect on our local field with lots of trainer Spektrum TX's available for wireless trainer.

**Comment by martingro85** [comment]
Bought the X18 and the 4in1 module because I never thought that this feature might not be available. It would be great if this could be available soon.
